services/web/server/src/simcore_service_webserver/login/_2fa.py

```diff
@@ -80,6 +80,7 @@ async def send_sms_code(
     twilo_auth: TwilioSettings,
     twilio_messaging_sid: str,
     product_display_name: str,
+    user_name: str = "user",
```

Comment:
ok, so same question, why do you set a default value although this is a private module?

Reply:
i would like to listen to your rationale about this point :-)